src/print.rs

```diff
@@ -104,50 +104,38 @@ pub fn explain(args: ArgMatches) {
             let value = module.get_segments().join("");
             ModuleInfo {
                 value: ansi_term::ANSIStrings(&module.ansi_strings()).to_string(),
-                value_len: value.chars().count() + count_wide_chars(&value),
+                value_len: UnicodeWidthStr::width(value.as_str()),
                 desc: module.get_description().to_owned(),
             }
         })
         .collect::<Vec<ModuleInfo>>();
 
-    let mut max_ansi_module_width = 0;
-    let mut max_module_width = 0;
-
-    for info in &modules {
-        max_ansi_module_width = std::cmp::max(
-            max_ansi_module_width,
-            info.value.chars().count() + count_wide_chars(&info.value),
-        );
-        max_module_width = std::cmp::max(max_module_width, info.value_len);
-    }
+    let max_module_width = modules.iter().map(|i| i.value_len).max().unwrap_or(0);
 
     let desc_width = term_size::dimensions()
         .map(|(w, _)| w)
-        .map(|width| width - std::cmp::min(width, max_ansi_module_width));
+        .map(|width| width - std::cmp::min(width, max_module_width + 6));
```

Comment:
I think it makes sense to fall back to `None` instead of allowing only one char per line. Something like 4 could be a good  cutoff. Any opinions on this?

Reply:
Also should I replace the 6 with a 7 so there's padding on both the left and right side?

Reply:
I've tested this locally and 6 LGTM, only suggestion here would be to maybe add a comment to explain this so if we want to change it in the future it's nice an easy to find what we need to change.